src/sonic-ctrmgrd/setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup, find_packages
+
+setup_requirements = ['pytest-runner']
+
+test_requirements = ['pytest>=3']
+
+# read me
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+setup(
+    author="sonic-dev",
+    author_email='remanava@microsoft.com',
+    python_requires='>=3.6',
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+        'Natural Language :: English',
+        'Programming Language :: Python :: 3.6',
+    ],
+    description="Package contains remote container mgmt modules",
+    url='https://github.com/Azure/sonic-buildimage',
+    tests_require = [
+        'pytest',
+        'pytest-cov',
+    ],
+    install_requires=['netaddr', 'pyyaml'],
```

Comment:
Does this package really require `netaddr`?

Are there any other packages that you should list here?

Reply:
Yes it does require netaddr. Guess the rest are common ones.